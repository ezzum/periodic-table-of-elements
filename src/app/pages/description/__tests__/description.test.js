import React from 'react';
import { mount, shallow } from 'enzyme';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-redux';

import Description from '../description';

jest.mock('../../../exit-button/exit-button', () => 'div');

describe('Description tests', () => {
  const initialState = { activeElement: 'element' };
  const mockStore = configureStore();
  let store, container;

  beforeEach(() => {
    store = mockStore(initialState);
    // container = shallow(<Description store={store} />);
    container = mount(
      <Provider store={store}>
        <Description />
      </Provider>
    );
  });
  it('snapshot test', () => {
    expect(container).toMatchSnapshot();
  });
});
