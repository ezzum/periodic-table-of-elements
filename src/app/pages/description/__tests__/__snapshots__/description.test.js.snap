// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description tests snapshot test 1`] = `ReactWrapper {}`;
